fleurs-du-mal/FDM-091-119-000-144.xml

```diff
@@ -144,69 +123,105 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?>
       <body>
          <div type="poem">
             <head>
-               <title>Abel et Caïn</title>
+               <title type="main">ABEL ET CA<app>
+                     <lem wit="#U1857 #E1857">I</lem>
+                     <rdg wit="#FDM1861 #FDM1868">Ï</rdg>
+                  </app>N</title>
```

Comment:
I'd suggest taking a less granular approach to the annotations, e.g.
```xml
 <title type="main">ABEL ET 
    <app>
      <lem wit="#U1857 #E1857">CAIN</lem>
      <rdg wit="#FDM1861 #FDM1868">CAÏN</rdg>
   </app>
</title>

It may be difficult to index and display the changes if we take this fine an approach. Just my $.02.